hm_1.py

```diff
@@ -0,0 +1,5 @@
+num = input('Num: ')
+suma = 0
```

Comment:
Name of the variable is not so good.
Try to use PEP-8 suggestions during writing your code:
https://www.python.org/dev/peps/pep-0008/

Reply:
Renamed variables